en/docs/get-started/get-started.md

```diff
@@ -0,0 +1,474 @@
+
+10. Click **View Pull Request**.
+
+11. Review and click **Merge pull request**, and then click **Confirm Merge**. When you merge the pull request, it adds the necessary metadata files to connect your GitHub repository to Choreo so that you can proceed to create the component.
+
+
+###  Step 1.2: Deploy the REST API
+
+To deploy the REST API, follow the steps given below:
+
+1. Navigate back to the Choreo Console. You will be viewing the page for the `readingList` REST API.
+
+2. In the left pane, click the **Deploy** icon.
+
+3. Click **Deploy Manually**.
+
+    !!! info
+        Automatic deployment is enabled for the component by default. You are required to carry out only the first deployment manually. To learn more about automatic deployment, see [Configure Automatic Deployment](../tutorials/configure-automatic-deployment).
+
+   This deploys the API to the development environment to make it invokable.
+
+### Step 1.3: Test the REST API
+
+Let's test the `readingList` REST API via Choreo's Open API Console by following the steps given below:
+
+1. Click Test in the left pane.
+
+2. Expand the **POST** method.
+
+3. Click **Try it out**.
+
+4. Update the request body so that the parameters have the values given below:
+
+    | **Parameter** | **Value**       |
+    |---------------|-----------------|
+    | **author**    | `Bram Stoker`   |
+    | **status**    | `to_read`       |
+    | **title**     | `Dracula`       |
+
+    The request body should look as follows:
+
+    ```json
+      {
+        "author": "Bram Stoker",
+        "status": "To Read",
+        "title": "Dracula"
+      }
+    ```
+   
+5. Click **Execute**.
+
+    Check the **Server Response** section. You will have received a successful response with the `200` response code.
+
+Similarly, you can expand and try out the **GET** and **DELETE** methods.
+
+
+### Step 1.4: Publish the REST API
+
+In the previous step, you tested the `readingList` REST API and found that it works as expected. Therefore, let's publish it and make it available for subscriptions.
+
+#### Step 1.4.1: Update the CORS configuration
+
+The application developer in this scenario calls the API from a different domain (i.e., Vercel). By default, web browsers block these calls for security reasons. To enable the application to call the API, follow the steps given below:
+
+1. In the left pane, click the **Manage** icon.
+
+2. Click **Settings**.
+
+3. Under **API Settings** click **Edit**.
+
+4. Toggle the **CORS Configuration** switch to enable the CORS configuration.
+
+5. Select the **Access Control Allow Credentials** checkbox.
+
+6. In the **Apply to Development** pane that opens on the right of the page, enter a meaningful message. Then click **Apply**.
+
+7. Click **Save**.
+
+#### Step 1.4.2: Publish the REST API
+
+To publish the REST API, follow the steps given below:
+
+1. In the **Manage** tab, click **Lifecycle**.
+
+2. Click **Publish** to publish the REST API to the Developer Portal.
```

Comment:
hearing about the Developer Portal for the first time.

Reply:
Updated the sentence.